cumulus_library/actions/importer.py

```diff
@@ -0,0 +1,81 @@
+import pathlib
+import zipfile
+
+import pandas
+
+from cumulus_library import base_utils, errors
+from cumulus_library.actions import cleaner
+from cumulus_library.template_sql import base_templates
+
+
+def create_table_from_parquet(archive, file, study_name, db, schema):
+    try:
+        parquet_path = pathlib.Path(archive.extract(file))
+        table_types = pandas.read_parquet(parquet_path).dtypes
+        remote_types = db.col_parquet_types_from_pandas(table_types.values)
+        s3_path = db.upload_file(
+            file=parquet_path,
+            study=study_name,
+            topic=parquet_path.stem,
+            force_upload=True,
+            remote_filename=parquet_path.name,
+        )
+        query = base_templates.get_ctas_from_parquet_query(
+            schema_name=schema,
+            table_name=parquet_path.stem.replace(".", "_"),
+            local_location=parquet_path.parent,
+            remote_location=s3_path,
+            table_cols=list(table_types.index),
+            remote_table_cols_types=remote_types,
+        )
+        db.cursor().execute(query)
+    finally:
+        parquet_path.unlink()
+
+
+def import_archive(
+    config: base_utils.StudyConfig, archive_path: pathlib.Path, args: dict
+):
+    """Creates a study in the database from a previous export"""
+
+    # Ensure we've got something that looks like a valid database export
+    if not archive_path.exists():
+        raise errors.StudyImportError(f"File {archive_path} not found.")
+    try:
+        archive = zipfile.ZipFile(archive_path)
+        files = archive.namelist()
+        files = [file for file in files if file.endswith(".parquet")]
+    except zipfile.BadZipFile as e:
+        raise errors.StudyImportError(
+            f"File {archive_path} is not a valid archive."
+        ) from e
+    if len(files) == 0:
+        raise errors.StudyImportError(
+            f"File {archive_path} does not contain any tables."
+        )
+    study_name = files[0].split("__")[0]
```

Comment:
nit: there isn't a sanity check that there actually is a `__` in that filename. Not super likely, but the surrounding code is pretty paranoid.

Reply:
what's a little more paranoia between friends? will add.